ARCHITECTURE.md

```diff
@@ -0,0 +1,56 @@
+# Architecture
+
+
+
+# Overview
+
+**`node-nikita-arch`** leverages our automation and deployment solution written in Nodejs, [Nikita](https://nikita.js.org/), to provide tools for automated installation of Arch Linux on a local or remote machine. Of note we provide here two main Nodejs modules, `bootstrap` and `system`, that encapsulate a set of Nikita actions to ***(i)*** setup disk partitions and mount them accordingly, ***(ii)*** install and configure mandatory system requirements and drivers and ***(iii)*** run services and install more specific softwares (e.g., Atom, Sublime Text, Java, ) based on user configuration.
```

Comment:
This is a site note, you shouldn't start with Nikita. It shall come later. What does nikita do, then cover the fact that it use Nikita and that a good understand of how it works will help

Reply:
It doesn't provide tools, it is a tool to install Arch Linux targetting a desktop environnement.

Reply:
There are too main commands, bootstrap and system, explain why and where in the source code

Reply:
Ok understood.